check.php

```diff
@@ -0,0 +1,20 @@
+<?php
+    $input=$_GET['input'];
+    $word=$_GET['word'];
+    $count=0;
+    $pos=null;
+    $wrong_guess=$_COOKIE["count"];
```

Comment:
I've asked you all to keep the PHP variable and function names camelCased.
Also the DB name, DB columns and table names underscore_separated.